package.json

```diff
@@ -108,6 +110,6 @@
     "rollup-plugin-istanbul": "^3.0.0",
     "rollup-plugin-node-resolve": "^4.2.3",
     "stylus": "^0.54.8",
-    "testcafe": "^1.9.4"
+    "testcafe": "^1.14.2"
```

Comment:
I upgraded testcafe primarily because it now provides a `t.scroll` and `t.scrollIntoView`, which I leveraged in my schema selection tests.